.github/scripts/toolchain/build-mingw.sh

```diff
@@ -16,20 +16,51 @@ if [ "$RUN_CONFIG" = 1 ] || [ ! -f "$MINGW_BUILD_PATH/Makefile" ] ; then
                 --enable-debug"
         fi
 
-        case $PLATFORM in
+        case "$ARCH" in
+            x86_64)
+                TARGET_OPTIONS="$TARGET_OPTIONS \
+                    --disable-lib32 \
+                    --enable-lib64 \
+                    --disable-libarm32 \
+                    --disable-libarm64"
+            ;;
+            aarch64)
+                TARGET_OPTIONS="$TARGET_OPTIONS \
+                    --disable-lib32 \
+                    --disable-lib64 \
+                    --disable-libarm32 \
+                    --enable-libarm64"
+                CFLAGS="$CFLAGS \
```

Comment:
should it be removed here?

Reply:
Yes, this is leftover of the previous state. Thanks for noticing.

Reply:
CFLAGS are used anyway in the configuration below. I expected it is not needed here anymore.